CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ Breaking changes:
 Features:
 
 - [#1291](https://github.com/rails-api/active_model_serializers/pull/1291) Add logging (@maurogeorge)
-- [#1225](https://github.com/rails-api/active_model_serializers/pull/1125) Better serializer lookup, use nested serializer when it exists (@beauby)
+- [#1225](https://github.com/rails-api/active_model_serializers/pull/1225) Better serializer lookup, use nested serializer when it exists (@beauby)
```

Comment:
Awesome, thanks! While you're in there, would you mind fixing

`[#tbd](https://github.com/rails-api/active_model_serializers/pull/tbd) Rename ArraySerializer to` ...

to refer to https://github.com/rails-api/active_model_serializers/pull/1251 ?